lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -120,6 +120,28 @@ ::mlir::LogicalResult mlir::tt::ttir::MatmulOp::verify() {
 }
 // ANCHOR_END: adding_an_op_matmul_ttir_verify
 
+::mlir::LogicalResult mlir::tt::ttir::Conv2dOp::verify() {
```

Comment:
ttir and ttnn verify methods are exactly the same, let's reuse the code.

Reply:
They are for now. There's still much work to be done to bringup conv2d e2e and they could end up being different.

Reply:
@rpavlovicTT I don't think we should reuse code across dialects in cases like these. As @LPanosTT mentioned, they might change in the future, but it also feels very "non mlir way" to reuse in this instance.

Reply:
I kinda agree with you guys, it's just if something changes you'd have to change both functions, and it's easy to miss one place. But I'm fine with leaving it as is.